proposals/dao/tip_121a.ts

```diff
@@ -0,0 +1,204 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  PcvStats,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { BigNumber } from 'ethers';
+import { getImpersonatedSigner, overwriteChainlinkAggregator } from '@test/helpers';
+import { TransactionResponse } from '@ethersproject/providers';
+
+const OLYMPUS_MULTISIG = '0x245cc372C84B3645Bf0Ffe6538620B04a217988B';
```

Comment:
Always in general in favour of putting these magic addresses in `mainnetAddresses.ts`

Reply:
done